release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -3650,6 +3650,13 @@ $ oc adm release info 4.14.3 --pullspecs
 [id="ocp-4-14-3-bug-fixes"]
 ==== Bug fixes
 
+* Previously, `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) objects could not be used if there were any `ImageContentSourcePolicy` (ICSP) objects in the cluster. As a consequence, to use IDMS or ITMS objects, you needed to delete any ICSP objects in the cluster, which required a cluster reboot. With this change, ICSP, IDMS, and ITMS objects now function in the same cluster at the same time. As a result, you can use any or all of the three types of objects to configure repository mirroring after the cluster is installed. For more information, see xref:../post_installation_configuration/preparing-for-users.html#images-configuration-registry-mirror_post-install-preparing-for-users[Understanding image registry repository mirroring]. (link:https://issues.redhat.com/browse/RHIBMCS-185[RHIBMCS-185])
++
+[IMPORTANT]
+====
+Using an ICSP object to configure repository mirroring is a deprecated feature. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
```

Comment:
Also up to you! 
```suggestion
Using an ICSP object to configure repository mirroring is a deprecated feature. Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in future releases and is not recommended for new deployments.
```

Reply:
@kelbrown20 This is standard wording in our docs and should probably be changed globally.